README.md

```diff
@@ -98,7 +134,24 @@ To reduce the size of the installation from about 800 MByte down to about 200 MB
    * try `npm run unlock` from project root before calling `[sudo] npm run link`
    * try `npm link` in each package directory in this order: td-tools, core, binding-\*, cli, demo-servients
 
-## No time for explanations - I want to start from something running!
+### As a browser library
+
+Node-wot can also be imported as browser-side library. To do so, include the following `script` tag in your html:
+```html
+<script src="https://cdn.jsdelivr.net/npm/@node-wot/browser-bundle@latest/dist/wot-bundle.min.js"></script>
```

Comment:
I assume this CDN links works once the wot-bundle is pushed on NPM, Correct?
Currently it fails.

Existing packages like https://cdn.jsdelivr.net/npm/@node-wot/td-tools@latest/dist/td-tools.min.js work.

Shall we add a note to readers?

Reply:
Yes, once it is on NPM it will work. This is actually the best approach in my opinion as the alternative would be to upload the compiled code on `/dist` to github and use the github link, which should be avoided, and it should not be a problem if a snapshot is released on NPM directly after this PR is merged.

I also think that it would be useful to have more SNAPSHOT releases available on NPM for node-wot, as currently anyone using NPM to get node-wot would be missing on all the changes made in the last 2 months.

The typescript-definitions for example are already on 0.7.0-SNAPSHOT-5.